File: Codeforces-768A.cpp
#include <iostream>
#include <cstring>
#include <vector>
#include <algorithm>
#include <iomanip>
#define ll long long
using namespace std;
 
int main() 
{
    int num;
    cin >> num;
    vector<ll>vec(num);
    for (int i = 0; i < num; i++) { cin >> vec[i]; }
   
    if (num > 2) {
        sort(vec.begin(), vec.end());
        int min = vec[0], max = vec[num - 1],res=0;
        for (int i = 0; i < vec.size(); i++) {
            if (vec[i] > min && vec[i] < max) { res++; }
        }
        cout << res;
    }
    else {
        cout << "0";
    }
 
 
    return 0;
}

File: Codeforces-768A.java


import java.util.ArrayList;
import java.util.Arrays;
import java.util.Scanner;


public class Main{
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        int n = in.nextInt();
        int[] arr = new int[n];
        for(int i=0; i<n; i++){
            arr[i]=in.nextInt();
        }

        if(n<=2){
            System.out.println("0");
        }else{
            Arrays.sort(arr);
            int min = arr[0],max=arr[n-1];
            int res=0;
            for(int i=1; i<n-1; i++){
                if(arr[i]>min&&arr[i]<max){
                    res++;
                }
            }
            System.out.println(res);
        }
        in.close();
    }
}